<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 127);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="button01">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>60</y>
      <width>80</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);
border: 3px solid;
border-color: rgb(0, 0, 255);
background-color: rgb(156, 212, 255);</string>
    </property>
    <property name="text">
     <string>Button01</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button02">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>190</y>
      <width>80</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 255);
border: 3px dotted;
border-color: rgb(0, 170, 0);
background-color: rgb(85, 255, 0);</string>
    </property>
    <property name="text">
     <string>Button02</string>
    </property>
   </widget>
   <widget class="QLabel" name="label01">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>310</y>
      <width>180</width>
      <height>180</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>HY엽서L</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 255, 127);
border: 5px double dotted;
border-color: rgb(255, 170, 0);
color: rgb(170, 0, 0);</string>
    </property>
    <property name="text">
     <string>안녕하세요!</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
